<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormInicio</class>
 <widget class="QMainWindow" name="FormInicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulario de Inicio</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="LabelTitulo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>651</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Ingrese los siguientes valores para poder iniciar con la simulación</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelAviones">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Aviones por aterrizar: </string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelEscritorios">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Escritorios disponibles:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelEstaciones">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Estaciones de servicio disponibles:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTurnos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Turnos:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonMenuP">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Iniciar Simulación</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TextAviones">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TextEscritorios">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TextEstaciones">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TextTurnos">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LabelCTurnos">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabelCAviones">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabelCEscritorios">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabelCEstaciones">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTNombreA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Nombre del avion:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTurnoActual">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Turno actual:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelVTurno">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Turno actual:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTTipoA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Tipo del avión:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTCantidadP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Cantidad de pasajeros:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTDesabordaje">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Turno de desabordaje:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelAvionesEspera">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Aviones en espera:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelInformacionA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Información del avión en area de desabordaje</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ListAvion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTTurnoM">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Turnos para mantenimiento:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelConsola">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Consola</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTurnoDesabordaje">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Turno actual:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelNombreAvion">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Turno actual:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTipoAvion">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Turno actual:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelCantidadPasajeros">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Turno actual:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTurnosMantenimiento">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Turno actual:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonSiguiente">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>269</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Siguiente Turno</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ListConsola">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>491</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 10pt &quot;Sans Serif&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonGraficas">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Modelo Grafico</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1037</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
